### ISSUE TITLE
seperate data for each chamber

### ISSUE THREAD

**Issue by kingpimpy** [appsample-com/spiral-abyss#95]
i think it would be more helpful/accurate to have said title

i dont think most people use a single team anymore for all 3 chambers and 9* clear it all

**Comment by FaZe-Goku** [comment]
i agree, especially this abyss, 12-1-2, 12-2-1 & 12-2-2 each requires specific characters in order to clear the challenge fast if you're not a whale of course

**Comment by soap231** [comment]
Agreed
